client-src/elements/chromedash-gate-column.js

```diff
@@ -235,10 +282,40 @@ class ChromedashGateColumn extends LitElement {
     `;
   }
 
+  renderControls() {
+    if (!this.user || !this.user.can_comment) return nothing;
+    // TODO(jrobbins): checkbox to also post to intent thread.
+
+    return html`
+    <sl-textarea id="comment_area" rows=2 cols=40
+      @sl-change=${this.checkNeedsSave}
+      @keypress=${this.checkNeedsSave}
+      placeholder="Add a comment"
+      ></sl-textarea>
+     <div id="controls">
+       <sl-button variant="primary"
+         @click=${this.handlePost}
+         ?disabled=${!this.needsSave}
+         size="small"
+         >Post</sl-button>
+     </div>
+    `;
+  }
+
+
   renderComments() {
     return html`
       <h2>Comments &amp; Activity</h2>
-      TODO(jrobbins): Comments go here
+      ${this.renderControls()}
+      <div id="comment-area">
```

Comment:
comment-area is a different id? how is it used?

Reply:
Removed.  I had added that `<div>` because I thought I might need it do add some padding, but it turned out that I only needed the padding on the `<p>` elements.